dm/master/scheduler/scheduler.go

```diff
@@ -724,14 +749,22 @@ func (s *Scheduler) RemoveSubTasks(task string, sources ...string) error {
 		return nil // no subtask need to stop, this should not happen.
 	}
 
+	release, err := s.subtaskLatch.tryAcquire(task)
+	if err != nil {
+		return terror.ErrSchedulerLatchInUse.Generate("RemoveSubTasks", task)
+	}
+	defer release()
+
 	// 1. check the task exists.
-	stagesM, ok1 := s.expectSubTaskStages[task]
-	cfgsM, ok2 := s.subTaskCfgs[task]
+	v1, ok1 := s.expectSubTaskStages.Load(task)
+	v2, ok2 := s.subTaskCfgs.Load(task)
```

Comment:
Maybe more understandable naming. For all `v`.

```suggestion
    stagesMapV, ok1 := s.expectSubTaskStages.Load(task)
	cfgsMapV, ok2 := s.subTaskCfgs.Load(task)
```

Reply:
they will be quickly converted to a meaningful name, just 7 lines below 😂

Reply:
I see. But it’s better to give it a meaningful name also here.